lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -55,8 +55,10 @@ def self.buffers
       @buffers ||= {
                      upsert: {},
                      delete: {},
-                     assignments: [],
-                     links: [],
+                     assignments: {},
+                     deassignments: {},
```

Comment:
anyone got a better name for these buffers?  both `unassignments` and `deassignments` and the link equivalents seem a little gross.